prisma/seed.js

```diff
@@ -0,0 +1,210 @@
+        participants.push(participant);
+    }
+    return participants
+}
+
+const createRounds = async (season) => {
+    const rounds = [];
+    for (let i = 0; i < MAX_ROUNDS_TO_GENERATE; i++) {
+        const fakeRound = createFakeRound();
+        const round = await prisma.round.create({
+            data: {
+                title: fakeRound.title,
+                startsAt: fakeRound.startsAt,
+                season: {
+                    connect: {
+                        id: season.id
+                    }
+                }
+            }
+        })
+        console.log("round", round);
+        rounds.push(round);
+    }
+    return rounds;
+
+}
+
+const createPlacements = async (participants, rounds) => {
+    const placements = [];
+
+    for (let i = 0; i < rounds.length; i++) {
+        const positions = Array.from({ length: participants.length }, (_, i) => i + 1); //fills an array up to max of participants [1,2,3 ... participants.length]
+        for (let j = 0; j < participants.length; j++) {
+            const fakePosition = createFakePlacement(positions);
+            const placement = await prisma.placement.create({
+                data: {
+                    position: fakePosition.position,//add fake position
+
+                    participant: {
+                        connect: {
+                            id: participants[j].id
+                        }
+                    },
+                    round: {
+                        connect: {
+                            id: rounds[i].id
+                        }
+                    },
+                }
+            });
+            const index = positions.indexOf(fakePosition.position);
+            positions.splice(index, 1);
+            console.log("placement", placement);
+            placements.push(placement);
+        }
+    }
+    return placements
+}
+
+const createPositionMappings = async (season, participants) => {
+    const positions = Array.from({ length: participants.length }, (_, i) => i + 1); //fills an array up to max of participants [1,2,3 ... participants.length]
+    const fakeMappings = createFakePositionMapping(positions);
+    const positionMappings = [];
+    for (let i = 0; i < fakeMappings.length; i++) {
+        const positionMapping = await prisma.positionMapping.create({
+            data: {
+                position: fakeMappings[i].position,
+                mapping: fakeMappings[i].mapping,
+                season: {
+                    connect: {
+                        id: season.id
+                    }
+                }
+            }
+        });
+        console.log("positionMapping", positionMapping);
+        positionMappings.push(positionMapping);
+    }
+    return positionMappings
+}
+
+
+const seed = async () => {
+    const admins = await createAdmin();
+    for (let i = 0; i < admins.length; i++) {
+        for (let j = 0; j < admins[i].competition[0].seasons.length; j++) {
```

Comment:
Make a note of this, it'll be good to revisit what you've done here after you've been through the computer science week - specifically time complexity / "Big O notation"